CSFCamera.py

```diff
@@ -3,6 +3,8 @@
 from picamera import PiCamera
 import RPi.GPIO as GPIO
 import argparse as argp
+import os
+
 
 GPIO.setmode(GPIO.BCM)
```

Comment:
You should wrap the code in and `if __name__ == '__main__':` so that it does not if it gets imported by another module.

https://www.freecodecamp.org/news/if-name-main-python-example/

Reply:
Done